okio/src/jvmMain/kotlin/okio/Path.kt

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2020 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package okio
+
+import okio.internal.commonCompareTo
+import okio.internal.commonEquals
+import okio.internal.commonHashCode
+import okio.internal.commonIsAbsolute
+import okio.internal.commonIsRoot
+import okio.internal.commonNameBytes
+import okio.internal.commonParent
+import okio.internal.commonResolve
+import okio.internal.commonToPath
+import okio.internal.commonVolumeLetter
+
+@ExperimentalFileSystem
+actual class Path internal actual constructor(
+  internal actual val slash: ByteString,
+  internal actual val bytes: ByteString
+) : Comparable<Path> {
+  actual val isAbsolute: Boolean
+    get() = commonIsAbsolute()
+
+  actual val isRelative: Boolean
+    get() = !isAbsolute
```

Comment:
My preference would be to have common methods for these as well (`isRelative`, `name`, `toString`), even though they're just one-liners still feels like code duplication.

Reply:
Done!